web/js/modules/image-download/util.js

```diff
@@ -287,3 +289,53 @@ export function getPercentageFromPixel(maxDimension, dimension) {
 export function getPixelFromPercentage(maxDimension, percent) {
   return Math.round((percent / 100) * maxDimension);
 }
+
+/**
+ * Find if there are layers that cannot be downloaded
+ * @param {Array} visibleLayers
+ *
+ * @return {Bool}
+ */
+export function hasNonDownloadableVisibleLayer(visibleLayers) {
+  const notDownloadable = !!lodashFind(visibleLayers, { snapshot: false });
```

Comment:
Not really a big deal either way but, why use lodash when we can do this natively with ES6?  I don't think we need to phase out lodash entirely since it does have some handy methods that don't exist natively and the file size savings would only be around 25kb gzipped but, I've been trying to replace things like this when I see them since I've got a slight preference for the native approach.  For example, using the spread operator in a reducer instead of lodash assign just looks cleaner to me.

Doing a find would be just a simple one liner in ES6 as well but, I think `Array.some()` is even more appropriate here since it returns a boolean:
```
export function hasNonDownloadableVisibleLayer(visibleLayers) {
  return visibleLayers.some(({ snapshot = true }) => !snapshot);
}
```